packages/liveblocks-client/src/room.ts

```diff
@@ -1724,7 +1724,9 @@ function prepareCreateWebSocket(
   const ws = WebSocketPolyfill || WebSocket;
 
   return (token: string): WebSocket => {
-    return new ws(`${liveblocksServer}/?token=${token}`);
+    return new ws(
+      `${liveblocksServer}/?token=${token}&version=__PACKAGE_VERSION__`
```

Comment:
Oh, just thought about this… perhaps in non-builds (aka local dev), let this string default to `"dev"`?

Reply:
Tackled in #368.